cli/azd/pkg/project/resource_manager.go

```diff
@@ -143,7 +143,7 @@ func (rm *resourceManager) GetServiceResource(
 		if len(resources) == 0 {
 			err := fmt.Errorf(
 				//nolint:lll
-				"unable to find a resource tagged with '%s: %s'. Ensure the service resource is correctly tagged in your bicep files, and rerun %s",
+				"unable to find a resource tagged with '%s: %s'. Ensure the service resource is correctly tagged in your infrastructure templates, and rerun %s",
```

Comment:
Is there a better wording than "infrastructure templates"? I worry this is confusing language.

It seems that bicep files is what `bicep` refers to it's configuration, [doc](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/overview?tabs=bicep).

And terraform refers to their files as Terraform configuration, [doc](https://developer.hashicorp.com/terraform/tutorials/cli/init#reinitialize-configuration)?

Reply:
I like the term "infrastructure configuration" for what it's worth.  I think that's generic enough to cover bicep and terraform (and others later) and doesn't have some of the connotations that template brings.